_health-care/_js/components/SubmitButton.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A component to submit the form.
+ *
+ * Required props
+ * `onButtonClick`: function that submits the form data.
+ */
+class SubmitButton extends React.Component {
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  render() {
+    return (
+      <div className="row progress-buttons">
+        <div className="small-8 columns">
+          <button
+              className="usa-button-primary"
+              id={`${this.id}-submitButton`}
+              onClick={this.props.onButtonClick}>Submit Application</button>
+        </div>
+      </div>
+    );
+  }
+}
+
```

Comment:
I am building out a ProgressButton component so that all buttons can use the same code and we will pass `onClick`, `buttonText` and `buttonClass` as props.